source/extensions/tracers/xray/tracer.cc

```diff
@@ -19,6 +19,7 @@ namespace XRay {
 
 namespace {
 constexpr auto XRaySerializationVersion = "1";
+constexpr auto DirectionKey = "direction";
```

Comment:
nit: `constexpr absl::string_view` would be better then `constexpr auto`.

Reply:
Made this as `constexpr absl::string_view`